companion/src/CMakeLists.txt

```diff
@@ -300,12 +300,19 @@ endif()
 
 ############# Install ####################
 
+# Generate list of simulator plugins, used by all platforms
+file(GLOB simulator_plugins "${CMAKE_BINARY_DIR}/*opentx-*simulator${CMAKE_SHARED_LIBRARY_SUFFIX}")
+# the current flavour is not automatically added if build in the current cmake iteration, so always
+# add its library name to be sure
+string(TOLOWER ${PCB} FLAVOUR)
+get_property(current_plugin TARGET opentx-${FLAVOUR}-simulator PROPERTY LOCATION)
+list(APPEND simulator_plugins "${current_plugin}")
```

Comment:
I don't like this hack. Do similar to the https://github.com/opentx/opentx/blob/c78ab55696ce5fc09726290d32d3ed913e25e579/launchpad/common/rules

 * make all simulator libs
 * run empty cmake `cmake <src dir>`  **this will detect the last simulator library**
 * then run `make package` or `make install`